osler/inventory/tests.py

```diff
@@ -0,0 +1,110 @@
+from django.test import TestCase
+from django.urls import reverse, resolve
+from osler.inventory.models import DrugCategory, MeasuringUnit, Manufacturer, Drug
+from osler.inventory import views, forms
+# Create your tests here.
+
+class TestUrls(TestCase):
+
+    def test_drug_list_url(self):
+        path = reverse('inventory:drug-list')
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-list')
+
+    def test_drug_add_new_url(self):
+        path = reverse('inventory:drug-add-new')
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-add-new')
+
+    def test_pre_add_new_drug_url(self):
+        path = reverse('inventory:pre-drug-add-new')
+        self.assertEqual(resolve(path).view_name, 'inventory:pre-drug-add-new')
+
+    def test_predrug_select_url(self):
+        path = reverse('inventory:predrug-select')
+        self.assertEqual(resolve(path).view_name, 'inventory:predrug-select')
+
+    def test_drug_dispense_url(self):
+        path = reverse('inventory:drug-dispense')
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-dispense')
+
+    def test_drugupdate_url(self):
+        path = reverse('inventory:drug-update', kwargs={'pk':1})
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-update')
+
+def drug_dict():
+
+    drug_dict = {
+        'name': 'Somedrug',
+        'unit': MeasuringUnit.objects.create(name='someunit'),
+        'dose': 1000.0,
+        'stock': 10,
+        'expiration_date': '2040-01-01',
+        'lot_number': 'ABCDEFGH',
+        'category': DrugCategory.objects.create(name='somecategory'),
+        'manufacturer': Manufacturer.objects.create(name='somemanufacturer')
+    }
+
+    return drug_dict
+
+class TestModels(TestCase):
+
+    def test_drugcategory_creation(self):
+        category = DrugCategory.objects.create(name='somecategory')
+        assert isinstance(category, DrugCategory)
+        self.assertEqual(category.__str__(), category.name)
```

Comment:
instead of assertEqual we have now decided that doing assert category.__str__() == category.name is the better way (don't ask me why)

Reply:
Aren't they the same thing?